docker/cypress/integration/main.spec.js

```diff
@@ -106,4 +106,43 @@ describe('Visual Regression Example', () => {
       capture: "fullPage"
     });
   });
+
+  it('should not fail if ALLOW_VISUAL_REGRESSION_TO_FAIL is set', {
+    env: {ALLOW_VISUAL_REGRESSION_TO_FAIL: true}
+  },  () => {
+    if (Cypress.env('type') === 'base') {
+      cy.visit('/04.html');
```

Comment:
Replace `/04.html` with `../../web/04.html` as there's no more ngnix server running to serve those files, since Cypress can open them directly. Also a couple of more places to fix below

Reply:
I changed all occurrences of `*.html`  in  the corresponding tests to `../../web/*.html`